ros2topic/ros2topic/verb/bw.py

```diff
@@ -117,11 +127,16 @@ def get_bw(self):
             return None, None, None, None, None
         with self.lock:
             n = len(self.times)
-            tn = time.monotonic()
+            tn = self.clock.now()
             t0 = self.times[0]
+            if self.use_sim_time:
+                if tn <= t0:
+                    self.times = []
```

Comment:
Adding an extra warning log seems friendly to users. I think it's mentioned [before](https://github.com/ros2/ros2cli/pull/754#discussion_r966636987).